docs/rackhd2.0/configuration.rst

```diff
@@ -15,6 +15,7 @@ config.json_
         "apiServerAddress": "172.31.128.1",
         "apiServerPort": 9080,
         "broadcastaddr": "172.31.131.255",
+        "dhcpPollerActive": true,
```

Comment:
We don't particularly care which is default, so long as the option is there. arp poller does not work with routed vlans (dhcp helper/forwarding) and it has problems when an ip address changes, because then you end up with duplicate arp entries.

https://github.com/RackHD/RackHD/issues/225